index.html

```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
 
             <h2 class="h2">Get Started</h2>
             <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-            <form action="" class="contact__form">
-                <input type="text" placeholder="your name" name="name" class="name">
-                <input type="email" placeholder="your email" name="email" class="email">
-                <input type="text" placeholder="Enter your message here..." name="message" id="" class="message">
-                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            <form action="https://formspree.io/f/mrgdozvp" method="post" class="contact__form">
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+                <textarea type="text" required placeholder="Enter your message here..." name="message" id="" class="message"> </textarea>
+                <input type="submit" maxlength="500" value="Start Collaboration" class="buttons contact__button">
```

Comment:
- [ ] This maxlength attribute should be used in the textarea, not the submit button as we are not writing anything in the submit.